quic/client/handshake/ClientHandshakeFactory.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+#pragma once
+
+#include <memory>
+
+namespace quic {
+
+class ClientHandshake;
+class QuicCryptoState;
```

Comment:
This is actually a `struct` right?

Reply:
Good catch. Considering this do not mangle the same I'm surprised this builds at all.